main.tf

```
@@ -9,20 +9,20 @@ resource "random_string" "this" {
 }
 
 module "equinix-fabric-connection" {
-  source = "github.com/equinix-labs/terraform-equinix-fabric-connection"
+  source  = "github.com/equinix-labs/terraform-equinix-fabric-connection"
```

Test complains on that source format, it should be:

```suggestion
  source  = "equinix-labs/fabric-connection/equinix"
```

---

Please update also version = "0.3.1" to version = "0.4.0"

---

updated